tests/nnstreamer_filter_extensions_common/unittest_tizen_template.cc.in

```diff
@@ -148,6 +148,8 @@ TEST (nnstreamer_@EXT_ABBRV@_basic_functions, get_dimension_fail_n)
   GstTensorsInfo res;
   gchar **model_files;
 
+  gst_tensors_info_init (&res);
```

Comment:
Can we simply do GstTensorsInfo res = {0}; ?

Reply:
Yes we can do like that because we know its internal impl. of initialization. But I think we need to consider possible changes of the initialization impl in future. So, I think it's safe to use the official function to initialize.

Reply:
It is generally ok for test codes, but for capi-single code, the latency matters and I'd like to minimize latency overhead of invocation (under 0.1us)  for embedded devices. We didn't start serious performance optimization, yet, but we are going to squeeze out any unnecessary operations.

For data structure changes, we may need to add some proper warnings in data structure definitions (tensor_typedef.h). E.g., ```@caution This required to be initialized by ${type} val = {0};``` for GstTensorsInfo, GstTensorInfo, tensor_type, tensor_dim.

Reply:
Or.. you may define a macro
GstTensorsInfoDeclare0 (val);
Which does
GstTensorsInfo val = {0};
and potentially we can implement proper initialization code later if GstTensorsInfo gets expanded (I don't want to expand it, though).

Reply:
> It is generally ok for test codes, but for capi-single code, the latency matters

Yes, it makes sense. I'll update codes.

Reply:
I applied your suggestion after changing the enum definition of invalid tensor type.